### ISSUE TITLE
Add feedback to user that tests are running

### ISSUE THREAD

**Issue by HAHermsen** [tkucic/codesys_code_execution_speedTests#1]
See title

for example show runs to go and countdown of the 1mio counter..

**Comment by tkucic** [comment]
That's how codesys works, it's a feature and a big at the same time. These measurements take up 100% of CPU time and overshoot the cycle time. Because of this Codesys doesn't update the visualization as it's trying to finish the iec tasks first.
I'll see what I can do with this

**Comment by HAHermsen** [comment]
Agreed, the behaviour is by-design. Offcourse the real reason is "oversaturation" of the CPU within the task as this is the goal of the program. However it's not user friendly to show "nothing", but atleast give an indication of starting the test.

Suggested workaround => add a timer in your main program which counts down from 10s to 0 .after the timer becomes TRUE, youn could then trigger running the rest of the programs via a simple IF THEN. During this pre-amble you could paint the starting screen and announce the test is starting and will take a while to finnish or something similar.

**Comment by tkucic** [comment]
I made some updates to the projects visu and the programs. Try it out, let me know how it works for you.
